app/controllers/publishers/vacancies/publish_controller.rb

```diff
@@ -1,12 +1,22 @@
 class Publishers::Vacancies::PublishController < Publishers::Vacancies::WizardBaseController
+  # rubocop:disable Metrics/AbcSize
   def create
     if vacancy.published?
       redirect_to organisation_job_path(vacancy.id), notice: t("messages.jobs.already_published")
     elsif all_steps_valid? && PublishVacancy.new(vacancy, current_publisher, current_organisation).call
       update_google_index(vacancy) if PublishedVacancy.find(vacancy.id).live?
+
+      unless vacancy.contact_email_belongs_to_a_publisher?
+        Publishers::AccountInvitationMailer.invite_user(
+          contact_email: vacancy.contact_email,
+          publisher_email: current_publisher.email,
+        ).deliver_now
```

Comment:
Do we need this to be a `deliver_now` ? I think this blocks the request/response till the email is sent and can trigger an error if there is an issue with the mailer.
If we could just enqueue the email with a `deliver_later`, it would be non-blocking and faster for the user posting the vacancy.